gtests/net/mptcp/mp_join/mp_join_server_v4.pkt

```diff
@@ -17,22 +18,18 @@
 +0    accept(3, ..., ...) = 4
 
 // add_addr
-+0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
-// TODO: send echo bit
-// +0.0 <                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1],           dss dack8=1 dll=0 nocs>
-
-
-+0.2    <                               P.  1:3(2)  ack 1  win 256    <nop, nop,         TS val 4074418292 ecr 4074410674,                mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
-// MP_CAPABLE carrying data are acked with 64-bit, safer not knowing if the
-// sender will use a DSN on 64-bit or 32-bit. Later we can use the lower 32 bits
-+0.0    >                                .  1:1(0)  ack 3             <nop, nop,         TS val 4074418293 ecr 4074418292,                dss dack8=3 dll=0 nocs>
++0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address address_id=1 addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
++0.0    <                                .  1:1(0)  ack 1  win 256    <                                                    add_address address_id=1 addr[saddr1] addr_echo, dss dack8=1 dll=0 nocs>
```

Comment:
detail: we usually don't align the `<` that is used at the beginning of the TCP options → `<add_address`, not `< (...) add_address`
but we can change that if we prefer!

(same in `mp_prio/mp_prio_server_v4.pkt`)

Reply:
yes, correct